README.md

```diff
@@ -73,6 +74,9 @@ If you are troubleshooting realtime event handling in a perspective, add query p
 ## API Documentation
 The API is self-documenting based on [`./api/v1/swagger.yaml`](./api/v1/swagger.yaml). Start your server and open `MY_HOST:MY_PORT/v1/docs` for interactive documentation of all the Refocus API endpoints.
 
+## Refocus Rooms Documentation
+Get started with Refocus Rooms with the [User Guide](https://docs.google.com/document/d/1CV800MUPEVzf5_J2Age8CVbhn70waXxyDPgksLDLy0g/edit#heading=h.wirt8zok39rs).
```

Comment:
We can't link to a Salesforce google doc--this is a public repo. If the docs are public, add the docs as a page in the docs folder here in the repo.